Libplanet/Blocks/Block.cs

```diff
@@ -41,6 +42,12 @@ public class Block<T> : ISerializable
             Timestamp = timestamp;
             Transactions = transactions.ToArray();
```

Comment:
Is this property still required?

Reply:
I remained it to reduce a cost of the process to check tx nonces.
But In my thought, it can use a lot of memory.

Reply:
It will be great if `OrderedTransactions` is stored once it's calculated. IMO removing `Transactions` property can be delayed after then.